example/src/containers/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useState} from "react";
+import {Container, Button, Typography, Grid, Paper, Table, TableContainer, TableCell, TableRow, TableHead, TableBody, Hidden} from '@material-ui/core/';
+
+export const Dashboard = () => {
+    const [showElements, setShowElemnts]=useState<boolean>(false);
+
+    // TEMP
+    function createData(name: string, calories: number, fat: number, carbs: number, protein: number) {
```

Comment:
I think it would be better to generate more appropriate tx data:
timestamp, from address, to address, icon(outgoing/incomming), amount, token